lint/lint.go

```diff
@@ -15,19 +15,23 @@ type checkFunction interface {
 }
 
 type ruleInfo struct {
+	// SyntaxOnly marks rules that can be checker by using AST only.
+	// Such rule implementations should not use types info.
+	//
+	// It's OK not to mark rule as SyntaxOnly even if current
+	// implementation does not use types info.
+	SyntaxOnly bool
+
 	// Experimental marks rule implementation as experimental.
 	// Reasons to mark anything as experimental:
 	//	- rule name can change in near time
 	//	- checker gives more false positives than anticipated
 	//	- rule requires more testing or alternative design
 	Experimental bool
 
-	// SyntaxOnly marks rules that can be checker by using AST only.
-	// Such rule implementations should not use types info.
-	//
-	// It's OK not to mark rule as SyntaxOnly even if current
-	// implementation does not use types info.
-	SyntaxOnly bool
+	// VeryOpinionated marks rule as controversive for some audience and
```

Comment:
s/controversive/controversial/g (there is another case below)?

Reply:
indeed, my bad